CHANGELOG.md

```diff
@@ -2,14 +2,16 @@
 
 ## Features
 
+- Added the `pybamm.DiscreteTimeSum` expression node to sum an expression over a sequence of data times, and accompanying `pybamm.DiscreteTimeData` class to store the data times and values ([#4501](https://github.com/pybamm-team/PyBaMM/pull/4501))
+- Double-layer SEI models have been removed (with the corresponding parameters). All models assume now a single SEI layer. ([#4470](https://github.com/pybamm-team/PyBaMM/pull/4470))
```

Comment:
This entry is duplicated, since it is already in the breaking changes, can it be removed here

Reply:
Sorry, I thought it had to be in both!